server/src/main/java/org/elasticsearch/action/search/TransportSearchAction.java

```diff
@@ -399,16 +412,52 @@ public void onFailure(Exception e) {
                 String clusterAlias = entry.getKey();
                 boolean skipUnavailable = remoteClusterService.isSkipUnavailable(clusterAlias);
                 OriginalIndices indices = entry.getValue();
-                SearchRequest ccsSearchRequest = SearchRequest.subSearchRequest(parentTaskId, searchRequest, indices.indices(),
-                    clusterAlias, timeProvider.getAbsoluteStartMillis(), false);
-                ActionListener<SearchResponse> ccsListener = createCCSListener(clusterAlias, skipUnavailable, countDown,
-                    skippedClusters, exceptions, searchResponseMerger, totalClusters,  listener);
```

Comment:
To simplify things we could also disable the minimize roundtrip path if the emulation is enabled ?

Reply:
I'm afaraid I don't fully understand the proposal. Which code path do you think we can disable? Or do you mean disregarding the "minimize_roundtrip" parameter if the emulation is enabled entirely and thereby only covering one case and the corresponding codepath?

Reply:
> only covering one case and the corresponding codepath?

Yes sorry that wasn't clear.